services/mgmt/applicationinsights/src/package_2021_03_08_only/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod models;
+pub mod operations;
+#[allow(dead_code)]
```

Comment:
Since we generate packages per api-version, is this even useful i.e. for the user to be able to specify the api-version (as a string or constant)? Seems redundant given the intended use case with crate features.

Reply:
I opened up https://github.com/Azure/azure-sdk-for-rust/issues/563 to review API versioning.